com.github.zocker_160.SyncThingy.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.github.zocker_160.SyncThingy
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+rename-icon: logo-512
+rename-appdata-file: metainfo.xml
+rename-desktop-file: SyncThingy.desktop
+command: SyncThingy
+
+finish-args:
+    - --share=ipc
+    - --share=network
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --device=dri
+    - --filesystem=host
+    - --talk-name=org.kde.StatusNotifierWatcher
+    - --talk-name=org.freedesktop.Notifications
+    - --talk-name=org.freedesktop.portal.Background
```

Comment:
But do you actually access this D-Bus name somehow? I glanced over the code and I can't find it. You don't need it to just let it run in the background.

Reply:
I opted to comment it out, is that acceptable?